compliance_checker/cf/util.py

```diff
@@ -261,7 +262,7 @@ def _get(self, entrynode, attrname, required=False):
     def __init__(self, cached_location=None):
         if cached_location:
             try:
-                with open(cached_location, 'r') as fp:
+                with io.open(cached_location, 'r', encoding='utf-8') as fp:
```

Comment:
@lukecampbell do we really want to catch a generic `Exception` below? That was masking the real bug on Windows here preventing the user from using downloaded table.

Hopefully the code change here will fix it but maybe we should remove the `try`/`Exception` case and let it fail when it cannot read from the file.